lib/katello/tasks/clean_candlepin_orphaned_products.rake

```diff
@@ -0,0 +1,41 @@
+require File.expand_path("../engine", File.dirname(__FILE__))
+
+namespace :katello do
+  desc "Removes custom products (and their associated pools and subscriptions) from Candlepin that are not in Katello"
+  task :clean_candlepin_orphaned_products => ["dynflow:client", "environment"] do
+    User.current = User.anonymous_admin #set a user for orchestration
+    Dir.glob(Katello::Engine.root.to_s + '/app/models/katello/*.rb').each { |file| require file }
```

Comment:
Why is this required? It seems to be the cause of the enum error.

Reply:
I thought I removed that already! I bet you're right, that's making it try to load everything twice..

Reply:
It was in the `katello:reimport` task which I used as a template for this. I didn't really think much about it.